SitumWayfinding/Classes/WayfindingLibrary/SitumMapsLibrary.swift

```diff
@@ -204,7 +204,39 @@ import GoogleMaps
     public func setOnFloorChangeListener(listener: OnFloorChangeListener?) {
         delegatesNotifier.floorChangeDelegate=listener
     }
-    
+
+    /**
+     Sets a delegate that get notified when the map is ready to interact with and fully loaded.
+     
+     - parameter listener: OnMapReadyListener
+     */
+    public func setOnMapReadyListener(listener: OnMapReadyListener?) {
+        delegatesNotifier.mapReadyDelegate = listener
+    }
+
+    /**
+     Select a poi in the map
+     - parameters:
+       - poi: the SITPOI you want to select
+       - completion: callback called when operation complete either successfully or with an error
+     */
+    public func selectPoi(poi: SITPOI, completion: @escaping (Result<Void, WayfindingError>) -> Void) {
+        do {
+            try self.toPresentViewController?.select(poi: poi) {
+                completion(.success(()))
+            }
+        } catch {
+            completion(.failure(self.wayfindindError(from: error)))
+        }
+    }
+
```

Comment:
Se cadra aquí compensa tratar o erro de forma máis xenérica. É posible que mentres non se traten con detalle todolos erros que chegen do sdk nos compense propagalos para fora, polo menos para que o integrador teña unha pista de que está pasando e non reciba un unknown.
Creo que no selectPoi non aplica porque non fai falla facer descargas de rede, pero para o map ready cando se implemente o onError posiblemente sí aplicaría